src/programs/aws.ts

```diff
@@ -185,6 +185,9 @@ export const AWSProgram = (opts: OptionValues) => {
             DATABASE_PASSWORD: db?.masterPassword,
           };
 
+    const CONTAINER_SERVICE_URL = containerService.url.endsWith('/')
```

Comment:
Pulumi variables are not strings, so thats why it is failing on `tsc`. 

Something like `containerService.url.apply(v => v.endsWith('/') ? v.slice(0, -1) : v)` should work in this case

Reply:
Ah okay.  Thanks for that info! Made the change